src/main/java/de/tum/in/www1/artemis/web/rest/util/PageUtil.java

```diff
@@ -52,9 +50,16 @@ public static PageRequest createExamPageRequest(PageableSearchDTO<String> search
     }
 
     @NotNull
-    public static PageRequest createGradingScaleRequest(PageableSearchDTO<String> search) {
+    public static PageRequest createGradingScalePageRequest(PageableSearchDTO<String> search) {
         var sortOptions = Sort.by(GradingScale.GradingScaleSearchColumn.valueOf(search.getSortedColumn()).getMappedColumnName());
         sortOptions = search.getSortingOrder() == SortingOrder.ASCENDING ? sortOptions.ascending() : sortOptions.descending();
         return PageRequest.of(search.getPage() - 1, search.getPageSize(), sortOptions);
     }
+
+    @NotNull
+    public static PageRequest createCoursePageRequest(PageableSearchDTO<String> search) {
+        var sortOptions = Sort.by(Course.CourseSearchColumn.valueOf(search.getSortedColumn()).getMappedColumnName());
+        sortOptions = search.getSortingOrder() == SortingOrder.ASCENDING ? sortOptions.ascending() : sortOptions.descending();
+        return PageRequest.of(search.getPage() - 1, search.getPageSize(), sortOptions);
+    }
```

Comment:
Can you find a better way to do this which does not duplicate the logic? The only difference is the used enum here, right?

Reply:
I'll look into that thanks

Reply:
I'll address that in this folow-up PR: https://github.com/ls1intum/Artemis/pull/7913
Would be cool if you could look into it and tell me what you think of it.